index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <nav class="nav-container">
+            <ul class="nav-links">
+                <li class="logo" >
+                    <a href="#">My logo</a>
+                </li>
+                <li class="bars">
+                    <a href="#"><i class="fa fa-bars fa-2x" aria-hidden="true"></i></a>
+                </li>
+            </ul>
+    </nav>
```

Comment:
- [ ] Please place logo outside of `<nav>` tag, We should place only navbar items and menu options inside `<nav>`, and logo is not part of our navbar.